DOLDatabase/Tables/MerchantItem.cs

```diff
@@ -66,6 +67,20 @@ public string ItemTemplateID
 				}
 			}
 
+			[DataElement(AllowDbNull=false)]
+			public long Price

```

Comment:
Hello, thank you for your pull request.
I’m a bit puzzled about adding the price directly in the merchant item. This means that the price of an object no longer depends on the object itself but rather on the merchant. This implies that an object can potentially have a different price on different merchants. While I understand the design choice, this makes little sense on daoc to me.
Should we rather add an AltPrice column in the ItemTemplate instead? I think this has been proposed on the forums a while ago but never really followed up. Of course this does not solve the problem of the multi currency. There would be one column per currency? But this would strongly limit the number of currencies. Or maybe add an AltPrices (plural) column on the ItemTemplate as a string where you specify the alternative price of the object in the form of a (JSON?) dictionary of values?
Or finally, we could define a conversion price for all available currencies once and for all in the server properties and let the server convert the price from one currency to the other on each page. Although I think I saw something similar in the code already?
I have to admit this is not an easy one.

Reply:
There is no reason why the price needs to be part of ItemTemplate for each and every currency (or actually any other currency).

> While I understand the design choice, this makes little sense on daoc to me.

That's how Live does it anyway.

Reply:
That would be a strong reason indeed, but how can you be so sure that this is how live does it?

Reply:
> That would be a strong reason indeed, but how can you be so sure that this is how live does it? i.e. that the price is attached to the merchant and not to the item.

Actually, it's better this way, if you want to do an event with "promotional" items, you can just prepare a merchant list with decreased prices.
For management, it's a bit harder to maintain but often you have only one merchant list for each item so it should be not so bad...

Also, I don't think DOL should try to maintain an exact copy of DAOC, it should be able to do more customisations. If a feature can copy DAOC and do more should stay that way and not be restrained just because "it's not DAOC".